test/unit/range/views/view_minimiser_test.cpp

```diff
@@ -25,15 +27,43 @@ using seqan3::operator""_shape;
 using result_t = std::vector<size_t>;
 using iterator_type = std::ranges::iterator_t< decltype(std::declval<seqan3::dna4_vector&>()
                                                         | seqan3::views::kmer_hash(seqan3::ungapped{4})
-                                                        | seqan3::views::minimiser(5))>;
+                                                        | seqan3::views::minimiser(5,
+                                                            std::declval<seqan3::dna4_vector&>()
+                                                            | seqan3::views::complement | std::views::reverse
+                                                            | seqan3::views::kmer_hash(seqan3::ungapped{4})
+                                                            | std::views::reverse))>;
+using iterator_type_no_rev = std::ranges::iterator_t< decltype(std::declval<seqan3::dna4_vector&>()
+                                                               | seqan3::views::kmer_hash(seqan3::ungapped{4})
+                                                               | seqan3::views::minimiser(5))>;
```

Comment:
The default should be the other way around.

`iterator_type == minimiser_view with 1 range::iterator`
`two_ranges_iterator_type == minimiser_view with 2 ranges::iterator`